bin/build-packages.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+// find all the packages
+const glob = require('glob');
+const {execSync} = require('child_process');
+const path = require('path');
+
+glob( 'packages/*/package.json', ( err, matches ) => {
+	if ( err ) { console.error( err ); return; }
+
+	matches.forEach( match => {
+		const dir = path.dirname( match );
+		const inputDir = path.join( dir, 'src' );
+		const outputDir = path.join( dir, 'dist', 'esm' );
+		console.log( `Building ${ dir }...` );
+		execSync( `node ./node_modules/.bin/babel -d ${outputDir} ${inputDir}`, {
+			env: Object.assign( {}, process.env, { CALYPSO_CLIENT: 'true' } )
+		} );
+	 } );
+	 console.log('Done building packages');
+} );
```

Comment:
And alternative approach to this file would be adding `build` `scripts` to individual files' `package.json`s (which arguably keeps 'em a bit more self-contained), and for the app-level build step, [`lerna run build`](https://lernajs.io/#command-run).

Reply:
the thing i ran into early was that `npx babel ..` wouldn't find the babel sitting in the node_modules a directory up. We could just go after it directly (`node ../node_modules/.bin/babel ...`)...

Reply:
Could be resolved by [hoisting](https://github.com/lerna/lerna/blob/master/doc/hoist.md)?

> Binaries from those common packages are symlinked to individual package `node_modules/.bin` directories, so that `package.json` scripts continue to work unmodified.

Reply:
possibly, but it means a new thing to use and learn, plus hoisting seems to be somewhat experimental.